packages/scans/src/index.js

```diff
@@ -6,11 +6,16 @@ const _ = require('lodash');
 const actionHelper = require('./action-helper');
 
 let actionCommon = {
-    processReport: (async (token, workSpace, plugins, currentRunnerID, issueTitle, repoName) => {
+    processReport: (async (token, workSpace, plugins, currentRunnerID, issueTitle, repoName, allowIssueWriting = true) => {
         let jsonReportName = 'report_json.json';
         let mdReportName = 'report_md.md';
         let htmlReportName = 'report_html.html';
 
+        if (allowIssueWriting) {
```

Comment:
Was this meant to be negated?

Reply:
Not sure what you mean by negated? The other commenter pointed out there is a less verbose way to achieve this result, so we're adjusting to use that. I don't see anything being erased in this PR, but maybe I am hard of sight today!

Reply:
yes, the logic should be inverse, my bad.

Reply:
Oh! Lol. Sorry, I didn't even notice. My eyes just glanced over it. I will change that.

Reply:
@thc202 This was resolved, are you comfortable approving now?